src/OpenTelemetry/Metrics/MetricReader.cs

```diff
@@ -23,8 +23,9 @@
 namespace OpenTelemetry.Metrics
 {
     /// <summary>
-    /// MetricReader which does not deal with individual metrics.
+    /// MetricReader base class.
     /// </summary>
+    /// https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#metricreader
```

Comment:
Do we want this line? (it might not be a valid xml comment, which could confuse the parser?)